src/components/FilterPanel/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface IFilterPanel {
+  changeSearchCallback: (range: number[]) => void;
```

Comment:
1. Please don't invent your own bicycle. It should be named `onChange`.
2. The filters state should be an object. 

```ts
export type Range = [gte: number, lte: number]

export type FiltersValue {
   voteAverage?: Range
}
```

```tsx
const [filters, setFilters] = useState<FiltersValue>({})

return (
   <Filters value={filters} onChange={setFilters} />
)
```

```ts
type FiltersProps = {
   value?: FiltersValue
   onChange?: (value: FiltersValue) => void
}

function Filters(props: FiltersProps): ReactElement {}
```

The component of course should be controllable.